### PR TITLE
Allow specifying rollout strategy

### PR DESCRIPTION
**Pull request by ellie**
I'd like to test some changes on cloud, and we need to make this configurable for that to happen.

I've set the default values to be the kubernetes defaults, so this should be a no-op until other configuration occurs.

## Type of change
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Checklist
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: charts/posthog/templates/events-deployment.yaml [hunk 1]
```diff
@@ -14,6 +14,13 @@ spec:
   {{- if not .Values.events.hpa.enabled }}
   replicas: {{ .Values.events.replicacount }}
   {{- end }}
+
```
Comment (guidoiaquinti) [inline charts/posthog/templates/events-deployment.yaml:17]: Do we want to change events only? What about the others?

Comment (ellie) [reply charts/posthog/templates/events-deployment.yaml:17]: I'm only really thinking of configuring those two atm, but I guess we could add this to everything 🤔 Will add that now

Comment (guidoiaquinti) [reply charts/posthog/templates/events-deployment.yaml:17]: Ah 👍 . I saw the changes in `values.yaml` and I thought some files were missing from the git commit.

Comment (ellie) [reply charts/posthog/templates/events-deployment.yaml:17]: Yeah the plugins deployment is all fancy and templated, so it doesn't need as many file changes

**Review by danielxnj — APPROVED** [review]
LGTM

**Pull request merged**